Article/Article.js

```diff
@@ -1,26 +1,45 @@
 // Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
-    // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
-    // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
-    // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
-  }
+    constructor(domElement) {
+        // assign this.domElement to the passed in domElement
+        this.domElement = domElement;
+        // create a reference to the ".expandButton" class.
+        this.expandButton = this.domElement.querySelector('.expandButton');
+        // Stretch
+        this.closeButton = this.domElement.querySelector('.btn');
+        // Using your expandButton reference, update the text on your expandButton to say "expand"
+        this.expandButton.innerText = 'expand';
+        // Set a click handler on the expandButton reference, calling the expandArticle method.
+        this.expandButton.addEventListener('click', () => {
+            this.expandArticle();
```

Comment:
You _could_ lose the curly brackets here:
```javascript
this.expandButton.addEventListener('click', () => this.expandArticle());
```
The fact that you are using them shows that you have a good understanding of what's going on, but even better yet is realizing that nothing will be happening with the return value (`undefined`) here and it's absolutely fine to just implicitly return it for the sake of brevity.

Reply:
The same goes for the `closeButton` event listener callback.